draft-ietf-quic-transport.md

```diff
@@ -2883,16 +2886,21 @@ needing acknowledgement are received.  The sender can use the receiver's
 Strategies and implications of the frequency of generating acknowledgments are
 discussed in more detail in {{QUIC-RECOVERY}}.
 
+An endpoint that is only sending acknowledgements will not receive
+acknowledgments from its peer unless those acknowledgements are included in
+packets with ACK-eliciting frames.  A sender SHOULD bundle ACK frames with
+other frames when possible.
```

Comment:
Maybe instead:

> Though it sending an ACK frame in every packet can be inefficient, a sender SHOULD send ACK frames in ACK-eliciting packets that it sends.

Which leads to a difficult question: how often do you send an ACK frame when you aren't receiving ACK-eliciting packets?  We've agreed to make it possible to use the specs to build an "decently performant" stack.  Is this a point that isn't important, or is there a need for specific advice?

Reply:
> How often do you send an ACK frame when you aren't receiving ACK-eliciting packets?

The text above ("An endpoint MUST NOT send a packet containing only an ACK frame in response to a non-ACK-eliciting packet") suggests a resolute "never", unless you bundle it with some useless ACK-eliciting frame. (See my comment above.)

The text here is suggesting that when a stack is sending a packet with an ACK-ecliciting frame, it should include an ACK frame for the previously unacknowledged non-ACK-eliciting packet.

Reply:
Yes, that is always the problem case.